buttercup/cogs/helpers.py

```diff
@@ -48,10 +65,31 @@ def get_duration_str(start: datetime) -> str:
     return f"{duration_ms} ms"
 
 
-def get_progress_bar(count: int, total: int, width: int = 10) -> str:
+def get_progress_bar(

```

Comment:
Why is this edited?

Reply:
The command previously needed a progress bar, because it took ages to fetch the data. However, the rate command will probably need it again, so I left it in.